### PR TITLE
Add smoke test for logs signal

### LINKED ISSUE #5048: Add smoke test for logging signal
Now that we are updated to SDK 1.10.0, and have logging exporter autoconfiguration (https://github.com/open-telemetry/opentelemetry-java/pull/4015), we should add smoke test for logging signal to make sure that it's all working together.

### PR DESCRIPTION
**Pull request by breedx-splk**
Resolves #5048 

Intentionally left very simple. We can add additional scenarios/run configs as we see fit. Might also be nice to make some additional assertions around the data contents, especially the Resource.

### CODE REVIEW

**Comment by breedx-splk** [comment]
I just realized that we probably also want a GHA workflow for this? If so, I can add that here...

**Comment by breedx-splk** [comment]
> I just realized that we probably also want a GHA workflow for this? If so, I can add that here...

Just verified that it's already running OOTB inside the "other" suite.

File: smoke-tests/src/test/groovy/io/opentelemetry/smoketest/LogsSmokeTest.groovy [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  }
+
+  @Override
+  protected TargetWaitStrategy getWaitStrategy() {
+    return new TargetWaitStrategy.Log(Duration.ofMinutes(1), ".*Started SpringbootApplication in.*")
+  }
+
+  def "Should export logs"() {
+    setup:
+    startTarget(11)
```
Comment (trask) [inline smoke-tests/src/test/groovy/io/opentelemetry/smoketest/LogsSmokeTest.groovy:35]: probably worth running on 8, 11, 17 similar to `SpringBootSmokeTest`

**Review by trask — APPROVED** [review]
💯

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**